tests/console/yast2_vnc.pm

```diff
@@ -27,17 +28,24 @@ sub configure_remote_admin {
     send_key $remote_admin{allow_remote_admin_with_session};
     assert_screen 'yast2_vnc_allow_remote_admin_with_session';
     # Firewall Settings
-    if (check_screen 'yast2_vnc_firewall_port_closed') {
-        send_key $firewall_settings{open_port};
+    # bsc#1207390 skip open firewall ports for tumbleweed
+    if ((check_screen 'yast2_vnc_no_network_interface') && is_tumbleweed) {
+        send_key $cmd{next};
     }
-    # Firewall Details
-    assert_screen 'yast2_vnc_firewall_port_open';
-    send_key $firewall_settings{details};
-    assert_screen 'yast2_vnc_firewall_port_details';
-    send_key $firewall_details{network_interfaces};
-    assert_screen 'yast2_vnc_firewall_details_interface_selected';
-    send_key $cmd{ok};
-    assert_screen 'yast2_vnc_firewall_details_selected';
+    else {
+        if (check_screen 'yast2_vnc_firewall_port_closed') {
```

Comment:
perhaps would be better to not check twice (it is more expensive), but to check by both needles and then use assert_match to handle it.